recipes/cardiac-mps/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "cardiac-mps" %}
+{% set name_underscore = "cardiac_mps" %}
+{% set version = "2023.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name_underscore }}-{{ version }}.tar.gz"
+  sha256: f2f129468fce644a0e0ecfe45dcfd77d97615262e7e6b55d5b1fa665ac730396
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - "mps=mps.__main__:app"
+    - "mps2mp4=mps.__main__:run_mps2mp4"
+    - "mps-summary=mps.__main__:run_summary"
+    - "mps-analyze=mps.__main__:run_analyze"
+    - "mps-motion=mps.__main__:run_motion"
+    
+  script: python -m pip install -v --no-deps .
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - ap_features
+    - imageio
+    - imageio-ffmpeg
+    - matplotlib-base
+    - numpy
+    - scikit-image
+    - scipy
+    - tifffile
+    - typer
+    - xmltodict
+test:
+  imports:
+    - mps
+
+  requires:
+    - pip
+
+  commands:
+    - pip check
+    - mps --help
+
+about:
+  home: https://github.com/ComputationalPhysiology/mps
+  summary: Tools for working with cardiac mps files
+  license: LGPL-2.1-or-later
+  license_family: LGPL
+  license_file: LICENSE
```

Comment:
Need to include the licenses that are in the `licenses/` directory. You can use an expression (appropriate for all 3) like: `LGPL-2.1-or-later AND BSD-3-Clause AND BSD-2-Clause`

Reply:
Thanks for pointing that out. This should now be fixed in https://github.com/conda-forge/staged-recipes/pull/22473/commits/b39b66e53760eb6473bcc2151805444645a1fae8